[AutoPR Azure.ResourceManager.ComputeLimit]-generated-from-SDK Generation - .NET-5543509

azure-sdk opened the pull request:
Configurations:  'specification/computelimit/resource-manager/Microsoft.ComputeLimit/ComputeLimit/tspconfig.yaml', API Version: 2025-08-15, SDK Release Type: stable, and CommitSHA: '257777bd5dd2cec7c658f4517b030ae4d65e0812' in SpecRepo: 'https://github.com/Azure/azure-rest-api-specs' Pipeline run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=5543509 Refer to https://eng.ms/docs/products/azure-developer-experience/develop/sdk-release/sdk-release-prerequisites to prepare for SDK release.  **Release plan link:** [https://web.powerapps.com/apps/821ab569-ae60-420d-8264-d7b5d5ef734c?release-plan-id=f39bcd63-41b3-f011-bbd3-0022480bb79b](https://web.powerapps.com/apps/821ab569-ae60-420d-8264-d7b5d5ef734c?release-plan-id=f39bcd63-41b3-f011-bbd3-0022480bb79b) **Submitted by**: partsha@microsoft.com
## Release Plan Details
- Release Plan: https://aka.ms/sdk-release-planner?release-plan-id=f39bcd63-41b3-f011-bbd3-0022480bb79b
 Spec pull request: https://github.com/Azure/azure-rest-api-specs/pull/38491
Spec API version: 2025-08-15

azure-sdk committed:
Configurations:  'specification/computelimit/resource-manager/Microsoft.ComputeLimit/ComputeLimit/tspconfig.yaml', API Version: 2025-08-15, SDK Release Type: stable, and CommitSHA: '257777bd5dd2cec7c658f4517b030ae4d65e0812' in SpecRepo: 'https://github.com/Azure/azure-rest-api-specs' Pipeline run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=5543509 Refer to https://eng.ms/docs/products/azure-developer-experience/develop/sdk-release/sdk-release-prerequisites to prepare for SDK release.

ArcturusZhang commented on sdk/computelimit/Azure.ResourceManager.ComputeLimit/src/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,11 @@
+// Replace Microsoft.Test with the correct resource provider namepace for your service and uncomment.
+// See https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers
+// for the list of possible values.
+[assembly: Azure.Core.AzureResourceProviderNamespace("ComputeLimit")]
```
we should use the actual provider namespace here.

parthshah2305 committed:
Update Azure resource provider namespace to Microsoft.ComputeLimit

azure-sdk committed:
Configurations:  'specification/computelimit/resource-manager/Microsoft.ComputeLimit/ComputeLimit/tspconfig.yaml', API Version: 2025-08-15, SDK Release Type: stable, and CommitSHA: 'dbdbc75c548071c0f5f0bdfae18dd1b31a7ab95a' in SpecRepo: 'https://github.com/Azure/azure-rest-api-specs' Pipeline run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=5574767 Refer to https://eng.ms/docs/products/azure-developer-experience/develop/sdk-release/sdk-release-prerequisites to prepare for SDK release.

ArcturusZhang commented on sdk/computelimit/Azure.ResourceManager.ComputeLimit/README.md:
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure ComputeLimit management client library for .NET
+
+**[Describe the service briefly first.]**
```
Please also update the description here.

ArcturusZhang committed:
regenerate

ArcturusZhang committed:
switch to new generator

ArcturusZhang committed:
update changelog

ArcturusZhang committed:
update commit id

parthshah2305 committed:
Update CHANGELOG for version 1.0.0-beta.1 release date

parthshah2305 committed:
Clarify service description in README

Updated the README to provide a brief description of the Microsoft.ComputeLimit service.

parthshah2305 commented on sdk/computelimit/Azure.ResourceManager.ComputeLimit/CHANGELOG.md:
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Updated

ArcturusZhang committed:
Merge remote-tracking branch 'origin/main' into sdkauto/Azure.ResourceManager.ComputeLimit-5543509

ArcturusZhang committed:
regen

ArcturusZhang committed:
update inventory

ArcturusZhang committed:
Merge remote-tracking branch 'origin/main' into sdkauto/Azure.ResourceManager.ComputeLimit-5543509

parthshah2305 commented on sdk/computelimit/Azure.ResourceManager.ComputeLimit/api/Azure.ResourceManager.ComputeLimit.net8.0.cs:
```diff
@@ -0,0 +1,201 @@
+    public partial class ComputeLimitGuestSubscriptionCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.ComputeLimit.ComputeLimitGuestSubscriptionResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.ComputeLimit.ComputeLimitGuestSubscriptionResource>, System.Collections.IEnumerable
+    {
+        protected ComputeLimitGuestSubscriptionCollection() { }
+        public virtual Azure.ResourceManager.ArmOperation<Azure.ResourceManager.ComputeLimit.ComputeLimitGuestSubscriptionResource> CreateOrUpdate(Azure.WaitUntil waitUntil, string guestSubscriptionId, Azure.ResourceManager.ComputeLimit.ComputeLimitGuestSubscriptionData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
`guestSubscriptionId` can be a string in any of the accepted string formats for GUID

parthshah2305 replied:
We are currently following similar patterns as other related RPs using subscription Id which uses string type. For example - Mircosoft.Quota 

https://github.com/Azure/azure-sdk-for-net/blob/da29d9e3255ba1714171b7b7d42cfc0196fb4990/sdk/quota/Azure.ResourceManager.Quota/api/Azure.ResourceManager.Quota.net8.0.cs#L280

<img width="1957" height="248" alt="image" src="https://github.com/user-attachments/assets/8b386456-e981-432e-a1ee-611bd6ed885b" />

parthshah2305 commented:
@ArcturusZhang @live1206 if there are no more comments, can I get final approval please for the merge to go through?

jsquire reviewed (approved):
Inventory updates look good.
